src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -645,7 +645,13 @@ class misc_commandscript : public CommandScript
 
     static bool HandleDismountCommand(ChatHandler* handler, char const* /*args*/)
     {
-        Player* player = handler->GetSession()->GetPlayer();
+        Player* player = handler->getSelectedPlayerOrSelf();
+        if (!player)
```

Comment:
Doesn't the "false" on line 64 of this file mean that the dismount command can't be used from the console?